pvacseq/lib/main.py

```diff
@@ -143,6 +149,13 @@ def main(args_input = sys.argv[1:]):
     if args.fasta_size%2 != 0:
         sys.exit("The fasta size needs to be an even number")
 
+    if args.downstream_sequence_length == 'full':
+        downstream_sequence_length = None
+    elif args.downstream_sequence_length.isdigit():
+        downstream_sequence_length = args.downstream_sequence_length
+    else:
+        sys.exit("The downstream sequence length needs to be a positive integer or 'full'")
```

Comment:
I think what would happen is that the code in the downstream generate_fasta step would try to subset the downstream sequence string like so: `downstream_sequence[0:0]` (line 118 of generate_fasta.py). This is valid in python and returns an empty string which would result in no downstream sequence getting appended. Whether or not that is a logical thing to do is a different discussion.